cli_reference/cli_manager/cli-manager-release-notes.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-release-notes_{context}"]
+= {cli-manager} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-release-notes
+
+toc::[]
+
+The {cli-manager} allows you to install CLI plugins in both connected and disconnected environments.
+
+:FeatureName: Using the {cli-manager} to install and manage plugins for the OpenShift CLI
+include::snippets/technology-preview.adoc[]
+
+These release notes track the development of the {cli-manager} for {product-title}.
+
+//For an overview of the {cli-manager}, see xref   :../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {cli-manager}].
+
+[id="cli-manager-release-notes-0-1-0_{context}"]
+== {cli-manager} 0.1.0 Technical Preview
+
+Issued: 19 November 2024
+
+The following advisory is available for the {cli-manager} 0.1.0:
+//add correct link when available
+* link:https://access.redhat.com/errata/RHEA-2023:2035[RHEA-2023:2035]
```

Comment:
This is the advisory https://access.redhat.com/errata/RHEA-2024:8303 belongs to the latest 0.1.0 release.